### PR TITLE
TD 2659 - Updates to wizard and file uploader

### PR DESCRIPTION
**Pull request by jegasriskantha**
contributes to  [TD-2569](https://sce.myjetbrains.com/youtrack/issue/TD-2659/Create-Step-2)

## Changes

- Adds new error prop to FileUploader to allow setting the FileUploader in an error state. 

Updates request by @Sowbhagya-ipg  here https://github.com/IPG-Automotive-UK/virto/pull/685#issuecomment-2064006311
- Updates FileUploader height to 56px from 70px
- Updates padding on wizard action.

## Dependencies

builds on this PR: https://github.com/IPG-Automotive-UK/react-ui/pull/844 as I needed a single pre-release to use in current development. 

## UI/UX
Error on FileUploader and new height:
CleanShot 2024-04-18 at 21 39 03@2x

Padding on wizard actions: 
CleanShot 2024-04-18 at 21 41 38@2x

## Testing notes

Automated tests:
- Added a test to check that when the error prop is set on the FileUploader the error styling is present. 

Manual testing:
- Check that the error prop sets the error styling on the file uploader. 

## Author checklist before assigning a reviewer

- [x] Reviewed my own code-diff.
- [x] Branch has been run in docker.
- [x] PR assigned to me or an appropriate delegate.
- [x] Relevant labels added to the PR.
- [x] Appropriate tests have been added.
- [x] Lint and test workflows pass.

### CODE REVIEW

**Review by AronPetrauskas — COMMENTED** [review]
Manually tested `FileUploader` in storybook, Confirmed padding of `WizzardActions`.

Left one comment regarding the functionality of the `error` prop on `FileUploader`. The logic might create a confusing behavior, just want to make sure this is the intended functionality.

File: src/FileUploader/FileUploader.tsx [hunk 1]
```diff
@@ -57,7 +58,7 @@ export default function FileUploader({
   });
 
   // are we rendering an error state?
-  const isError = isDragReject || rejectionMessage;
+  const isError = isDragReject || rejectionMessage || error;
```
[the remark this answers is not in the document]
Comment (jegasriskantha) [reply src/FileUploader/FileUploader.tsx:61]: As discussed this is desired behaviour, but I have updated the comments to make this functionality a bit clearer.

**Review by AronPetrauskas — COMMENTED** [review]
Good documentation, concise and clear 👍

**Review by AronPetrauskas — DISMISSED** [review]

**Comment by Sowbhagya-ipg** [comment]
https://github.com/IPG-Automotive-UK/virto/pull/685#issuecomment-2064006311

I think there is a misunderstanding here @jegasriskantha The wizard I meant was actually the stepper on the top that has a bottom margin of 24px.  Attaching the image for ref.
Screenshot 2024-04-22 at 10 34 00 am

**Comment by jegasriskantha** [comment]
Reverted mistakingly changed margin, and updated the correct one: 
CleanShot 2024-04-23 at 12 02 30@2x

**Comment by AronPetrauskas** [comment]
Changes look good to me!

**Pull request merged**